components/colorslider/stories/template.js

```diff
@@ -0,0 +1,42 @@
+import { html } from 'lit-html';
+import { classMap } from 'lit-html/directives/class-map.js';
+import { Template as ColorHandle } from "@spectrum-css/colorhandle/stories/template.js";
+
+
+import '../index.css';
+import '../skin.css';
+
+export const Template = ({
+  rootClass = "spectrum-ColorSlider",
+  customClasses = [],
+  isDisabled = false,
+  isFocused = false,
+  vertical,
+  style = {},
```

Comment:
Is style still being used here as an input to color slider? I see colorHandleStyle being used but not where the style object is applied.